sympy/vector/tests/test_vector.py

```diff
@@ -51,6 +53,44 @@ def test_vector_sympy():
     assert v3 == v2
     assert v3.__hash__() == v2.__hash__()
 
+def test_kind():
+    assert C.i.kind is VectorKind()
+    assert C.j.kind is VectorKind()
+    assert C.k.kind is VectorKind()
+
+    assert Mul._kind_dispatcher(NumberKind, VectorKind()) is VectorKind()
+    assert Mul(2, C.i).kind is VectorKind()
+
+    v1 = C.x * i + C.z * C.z * j
+    v2 = C.x * i + C.y * j + C.z * k
+    assert v1.kind is VectorKind()
+    assert v2.kind is VectorKind()
+
+    assert (v1 + v2).kind is VectorKind()
+    assert Add(v1, v2).kind is VectorKind()
+    assert Cross(v1, v2).doit().kind is VectorKind()
+    assert VectorAdd(v1, v2).kind is VectorKind()
+    assert VectorMul(2, v1).kind is VectorKind()
+    assert VectorZero().kind is VectorKind()
+
+    assert v1.projection(v2).kind is VectorKind()
+    assert v2.projection(v1).kind is VectorKind()
+
+def test_vectoradd():
```

Comment:
There should be two blank lines between top-level functions, classes etc.

Reply:
One more blank line was added